docs/notebooks/thomson.ipynb

```diff
@@ -136,8 +136,9 @@
     "non_collective = {\n",
     "    'name': 'Non-Collective Regime', \n",
     "    'n': 5e15*u.cm**-3, \n",
-    "    'Te': 40*u.eV, \n",
+    "    'Te': np.array([40])*u.eV,\n",
     "    'Ti': np.array([10])*u.eV,\n",
+    "    'efract': np.array([1]), \n",
```

Comment:
I just realized that this is a bit concerning - if there's a single electron population, wouldn't it make sense to allow passing a scalar Te and skipping putting in efract?

Then we could simplify the notebook by putting scalar values here and calling `x.get('efract')` where we currently have `efract=x['efract']`. If we make `efract` default to `None`, then catch it as None and replace it with `np.ones(1)` in the function's body, it's gonna work nicely.

Come to think of it, we could - and should, because having mutable lists (arrays are probably the same thing) as arguments is a bad call - replace that for `ifract` as well.

Does that sound reasonable to you?

Reply:
@StanczakDominik Sure that makes sense: the first part of what you asked (scalar Te) was actually already implemented and I just forgot to update it in the notebook. I've changed that now.

I've also made the change you suggested for efract and ifract: they now default to None and in that case are replaced with np.ones(1). I change the notebook to use this capability as you suggested, too.

Reply:
Thank you! I'll look through tomorrow (SLEEP HYGIENE, I SHALL NOT ABANDON YOU FOR PURPOSES OF WATCHING COOL CODE) and probably merge soon :)